docs/_docs/snapshots/snapshots.adoc

```diff
@@ -76,6 +78,27 @@ snapshot data spread across the cluster. Each node keeps a segment of the snapsh
 The link:snapshots/snapshots#restoring-from-snapshot[restore procedure] explains how to tether together all the segments during recovery.
 ====
 
+== Incremental snapshots
+
+Low RPO (Recovery Point Object) value like a few minutes can be hardly achieved with full snapshots, since they require additional resources
+for creation and storing all partitions data. Instead, incremental snapshots can be used:
+
```

Comment:
In 1st and 3rd list items, we are talking about how and why snapshots can be used. The second list item ("consist of...") is rather the snapshot property. It is better to describe it separately. Depending on its importance, snapshots description can be turned on before or after the list. Agree with this?